content/changelog/2024-07-05.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Explore available Postgres extensions from console, more protection for default and protected branches, and lots of improvements and fixes.
+---
+
+### Explore Postgres extensions from the Neon Console
+
+Postgres extensions let you customize and extend your database's capabilities across a broad range of categories such as analytics, data transformation, full-text search, geospatial data processing, and more. Neon supports over 70 Postgres extensions.
+
+You can now explore all of Neon's supported extensions directly from the console. We've added an **Install Extensions** section to the **Quickstart**, which you can open from the Neon Console sidebar.
+
+![Quickstart extensions](/docs/relnotes/quickstart_extensions.png)
+
+Click **Install** to view the required `CREATE EXTENSION` command, and click **Docs** to view the extension's documentation.
+
+### More protection for default and protected branches
+
+To enhance the protection of [default](/docs/manage/branches#protected-branch) and [protected](/docs/guides/protected-branches) branches, we've disabled the **Delete** action for those branches.
+
+To delete a default branch, you must first set a different branch as the default. Before deleting a protected branch, you have to change its status to unprotected.
```

Comment:
> To delete a default branch, you must first set a different branch as the default.

This behaviour is not new. It used to work the same with primary branches.

Reply:
Thank you. I've removed reference to default branches.